### PR TITLE
Add register output format

### PR DESCRIPTION
**Pull request by kehh**

### CODE REVIEW

File: app/printTemplates/results/local/register.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+						$vn_placement_id,
+						array(
+							'forReport' => TRUE,
+							'purify' => TRUE
+						)
+					);
+					print (strlen($vs_display_value) > 100 ? strip_tags(substr($vs_display_value, 0, 97)) . "&hellip;" : $vs_display_value);
+					?></td><?php
+					$vn_line_count ++;
+					error_log($vs_display_value);
```
Comment (leftclickben) [inline app/printTemplates/results/local/register.php]: Your logs are showing ;-)

File: app/printTemplates/results/local/register.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+	<?php
+
+	$vo_result->seek(0);
+
+	$vn_line_count = 0;
+
+	while ($vo_result->nextHit()) {
+		if($vn_line_count === 0){
+			print "<thead><tr>";
```
Comment (leftclickben) [inline app/printTemplates/results/local/register.php]: I tend to prefer using either `?>html<?php` or `print "html"` (or `echo "html"`) consistently within the same file, but happy to merge this as-is if you don't want to change it.

File: app/printTemplates/results/local/register.php [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+			}
+			print "</tr></thead>";
+		}
+		?>
+		<tr class="row">
+				<?php
+
+				foreach ($va_display_list as $vn_placement_id => $va_display_item) {
+					?><td><?php
+					$bundle_class = preg_replace('/W+/', '_',$va_display_item['bundle_name']);
```
Comment (leftclickben) [inline app/printTemplates/results/local/register.php]: `/W+/g` or you'll only get the first one

Comment (leftclickben) [inline app/printTemplates/results/local/register.php]: Oh wait sorry this is php, my bad.  Ignore previous.

File: app/printTemplates/results/local/register.php [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+	$vo_result->seek(0);
+
+	$vn_line_count = 0;
+
+	while ($vo_result->nextHit()) {
+		if($vn_line_count === 0){
+			print "<thead><tr>";
+			foreach ($va_display_list as $vn_placement_id => $va_display_item) {
+				print "<th class='displayHeader'>" . $va_display_item['display'] . "</th>";
```
Comment (leftclickben) [inline app/printTemplates/results/local/register.php]: This is also generating inconsistent output wrt attribute value quoting style `'` here vs `"` below.  But again, just a style thing, it works either way, so I'm happy to ignore it.

**Comment by leftclickben** [comment]
The changes themselves look fine, apart from the `error_log`s (and style issues aside).  However I'm wondering if this is the place to put this.  We've got `providence-configuration` repo, we can put it in there, and symlink it, or better, if there is a constant for it, define the value in `setup.php` like we have with the config files (`__CA_LOCAL_CONFIG_DIRECTORY__`).

**Comment by kehh** [comment]
As discussed, there are a number of `local` directories (one for each directory in `app/printTemplates`). For simplicity sake we'll keep them in the RWAHS fork.

I have removed the logs and addressed your comments about tag styles.

**Pull request merged**